src/machines/java/com/enderio/machines/common/blockentity/StirlingGeneratorBlockEntity.java

```diff
@@ -36,10 +45,18 @@ public class StirlingGeneratorBlockEntity extends PowerGeneratingMachineEntity {
 
     public StirlingGeneratorBlockEntity(BlockEntityType<?> type, BlockPos worldPosition,
         BlockState blockState) {
-        super(CAPACITY, USAGE, type, worldPosition, blockState);
+        super(EnergyIOMode.Output, CAPACITY, FixedScalable.ZERO, type, worldPosition, blockState);
         addDataSlot(new FloatDataSlot(this::getBurnProgress, p -> clientBurnProgress = p, SyncMode.GUI));
     }
 
+    private int getBurnPerTick() {
+        return Math.min(1, BURN_SPEED.scaleI(this::getCapacitorData).get());
```

Comment:
What does the min do here, as less then 1 is 0?

Reply:
ah crap yea that should be a max, its just to stop the scale going to 0.

Reply:
Yeah that makes more sense